system/BaseModel.php

```diff
@@ -1678,12 +1678,13 @@ protected function objectToArray($data, bool $onlyChanged = true, bool $recursiv
      * @param bool          $onlyChanged Only Changed Property
      * @param bool          $recursive   If true, inner entities will be casted as array as well
      *
-     * @return array|null Array
+     * @return array Array with raw values.
      *
```

Comment:
```suggestion
```
I think the description here is not necessary, since the method has a return type.

Reply:
The intent of the comment is to indicate that the values are **raw values**. 
However, I doubt that most people correctly understand what raw values are.